### PR TITLE
To_Do_comments

### PR DESCRIPTION
**Pull request by sudhir-jedhe**

### CODE REVIEW

File: src/widgets/Menu.js [hunk 1]
```diff
@@ -39,12 +39,23 @@ export default class Menu extends createjs.Container {
...
+    } = this._label.getBounds();
+    const underLine = new createjs.Shape();
+    underLine.graphics.setStrokeStyle(3);
+    underLine.graphics.beginStroke('#FF4500');
+    underLine.graphics.moveTo(0, height + 2);
+    underLine.graphics.lineTo(width, height + 2);
+    underLine.graphics.endStroke();
+    this.addChild(underLine);
+    this.underline = underLine;
+    this.underline.visible = false;
```
Comment (mnewcomb314) [inline src/widgets/Menu.js]: This is underlining the whole label.  The desired behavior for addressing this todo is to only underline the first instance of a letter in the label that corresponds to the accessKey.

File: src/widgets/Menu.js [hunk 2]
```diff
@@ -125,9 +136,11 @@ export default class Menu extends createjs.Container {
 
   onFocus() {
     this._focusIndicator.visible = true;
+    this.underline.visible = true;
```
Comment (mnewcomb314) [inline src/widgets/Menu.js]: The indicator for the accessKey shouldn't be toggled in focus/blur handlers since that would mean a keyboard user needs to first tab to it to find out what the accessKey is which means the accessKey is not functioning as a shortcut.

File: src/widgets/MultiLineTextInput.js [hunk 3]
```diff
@@ -76,8 +76,6 @@ export default class MultiLineTextInput extends createjs.Container {
     this._height = height;
 
     this.addEventListener('click', this.onFocus);
-    // todo: handle mouse based blur causing event
-
```
Comment (mnewcomb314) [inline src/widgets/MultiLineTextInput.js:80]: Nit: technically the todo's in this file should remain since fixing text input by moving focus to the DOM translation element will be a separate PR.

File: src/widgets/SingleLineTextInput.js [hunk 4]
```diff
@@ -85,7 +85,6 @@ export default class SingleLineTextInput extends createjs.Container {
     this.accessible.placeholder = placeholderText;
 
     this.addEventListener('click', this.onFocus);
-    // todo: handle mouse based blur causing event
```
Comment (mnewcomb314) [inline src/widgets/SingleLineTextInput.js]: Nit: technically the todo's in this file should remain since fixing text input by moving focus to the DOM translation element will be a separate PR.

File: src/widgets/Menu.js (hunk 2 above)
Comment (sudhir-jedhe) [reply src/widgets/Menu.js]: removed

File: src/widgets/Menu.js (hunk 1 above)
Comment (sudhir-jedhe) [reply src/widgets/Menu.js]: updated

File: src/widgets/MultiLineTextInput.js (hunk 3 above)
Comment (sudhir-jedhe) [reply src/widgets/MultiLineTextInput.js:80]: reverted

File: src/widgets/SingleLineTextInput.js (hunk 4 above)
Comment (sudhir-jedhe) [reply src/widgets/SingleLineTextInput.js]: reverted

File: src/widgets/AppWindow.js [hunk 5]
```diff
@@ -772,13 +754,10 @@ export default class AppWindow extends createjs.Container {
     alert.addChild(alertLabel);
     alert.visible = false;
 
+    label = new createjs.Text('', '14px Arial');
```
Comment (mnewcomb314) [inline src/widgets/AppWindow.js:757]: Since the creation of the instance is moved outside the callback, please move the register call for this label and the associated addChild calls to right after this line instead of in the callback.  Otherwise I see errors in the console when submitting multiple times.

**Review by mnewcomb314 — APPROVED** [review]
